robocat/cadmin/100-namespace.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: tekton-pipelines
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: eu-geo # Used for Minio
```

Comment:
Is it a random name ?

Reply:
It is so that the minio s3 storage is available at `s3.eu-geo.svc.cluster.local`, which is the format gsutil likes

Reply:
ah I see :+1: make sense :wink: